tests/test_analysis_autovot.py

```diff
@@ -0,0 +1,22 @@
+from conch.analysis.autovot import AutoVOTAnalysisFunction
+import librosa
+from statistics import mean
+import wave
+import pytest
+from conch.analysis.segments import SegmentMapping
+from conch import analyze_segments
+
+
+def test_autovot(autovot_filenames, autovot_markings, autovot_correct_times):
+    mapping = SegmentMapping()
+    for filename in autovot_filenames:
+        with wave.open(filename,'r') as f: 
+            length = f.getnframes() / float(f.getframerate())
+        mapping.add_file_segment(filename, 0, length, channel=0, vot_marks=autovot_markings[filename])
+    func = AutoVOTAnalysisFunction(classifier_to_use="autovot/experiments/models/bb_jasa.classifier", window_min=-30, window_max=30, min_vot_length=5, max_vot_length=100)
```

Comment:
Looks like we would need the classifier as well.  Regardless we only really need 3 files from the autovot repo (the classifier, a wav and a textgrid).  I don't know exactly how autovot gets built/installed, but presumably the auto_vot_decode.py is going to have to be on the system path somewhere?